workbook/components/session_service.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from django.utils import timezone
+
+from workbook.models.models import Session
+
+
+class SessionService:
+    def get_time_difference(self, session_start_time):
```

Comment:
this is a general function we can use it through out the class and codebase, we can create a file called helper.py or util.py and put functions like this there.
also no need to have the param as session_start_time and maybe a good idea to return it as seconds to fall into division special cases and etc, just return in seconds and compare to seconds and if you want to compare it to 15 minute or something just also change that to seconds.
```suggestion
    def get_time_difference(self, start_time):
```